src/modules/i18n/index.yml

```diff
@@ -18,7 +18,8 @@
   contribution: "contribution"
   contributions: "contributions"
   noPosts: ":( pas d'article"
-  pageActions: "Action sur cette page"
+  BeTheFirstToContribute: "Soyez le premier à contribuer à cette page"
```

Comment:
Since there is key that lead to uppercased sentences and lower case (part of) sentences, I was thinking it can be a good idea to add this noticeable change of case.